docs/troubleshooting.md

```
@@ -0,0 +1,121 @@
+    }
+  }
+}
+```
+
+Add any directories containing tools your setup needs (e.g., kubectl, docker) to the PATH.
+
+## `uvx` Blocks uv Cache Cleaning
+
+**Problem:** If you're using `uvx unblu-mcp` instead of the recommended `uv tool install`, tools like `topgrade` or `uv cache prune` will hang with:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)